internal/pkg/api/metrics.go

```diff
@@ -78,6 +73,11 @@ func init() {
 	cntFileDeliv.Register(routesRegistry.newRegistry("deliverFile"))
 	cntGetPGP.Register(routesRegistry.newRegistry("getPGPKey"))
 	cntAuditUnenroll.Register(routesRegistry.newRegistry("auditUnenroll"))
+
+	err := report.SetupMetrics(logger.NewZapStub("instance-metrics"), build.ServiceName, version.DefaultVersion, registry.registry, registry.registry)
```

Comment:
@kruskall, you introduced the new signature here, can we use the same registry for both args here?